zanata-model/src/main/java/org/zanata/model/HAccount.java

```diff
@@ -108,7 +102,7 @@ public HPerson getPerson() {
     }
 
     @NaturalId
-    @UserPrincipal
+//    @UserPrincipal
```

Comment:
I assumed it was in case we switched to a more generic solution (driven by
annotations) again in future, to make it easier to put in some annotations
again.

On 7 August 2015 at 13:42, Damian Jansen <notifications@github.com> wrote:

> In zanata-model/src/main/java/org/zanata/model/HAccount.java
> <https://github.com/zanata/zanata-server/pull/908#discussion_r36488926>:
>
> > @@ -108,7 +102,7 @@ public HPerson getPerson() {
> >      }
> >
> >      @NaturalId
> > -    @UserPrincipal
> > +//    @UserPrincipal
>
> Was there a reason to keep this/these?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/908/files#r36488926>.
>



-- 

-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat